runatlantis.io/docs/custom-workflows.md

```diff
@@ -420,7 +420,7 @@ Or a custom command
 | run | string | none    | no       | Run a custom command |
 
 ::: tip Notes
-* `run` steps are executed with the following environment variables:
+* `run` steps in the main `workflow` are executed with the following environment variables:
```

Comment:
is the `main workflow` the right sentence to use here?

main workflow = non `pre` or `post` workflow right?

I'm just wondering if we refer it the same way in other parts of the docs

Reply:
> main workflow = non pre or post workflow right?

That was my thinking, yes.

Reply:
I think we should say 
```
steps are executed with the following environment variables 
* note: these variables are not available to `pre` or `post` workflows
```